cal/cal/migrations/0046_auto_20170107_2152.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
You need migrations whenever you change database schemas. Otherwise you would pull this branch and your app would break because the code is looking for a database table that doesn't exist